src/platforms/apple/common/performance/instrumentation/swiftui-instrumentation.mdx

```diff
@@ -0,0 +1,79 @@
+---
+title: SwiftUI Instrumentation
+sidebar_order: 12
+description: "Learn how to monitor the performance of your SwiftUI views."
+---
+
+<Include name="beta-note.mdx" />
+
+You can monitor the performance of your views in a SwiftUI project with SentrySwiftUI library.
+
+Install it with CocoaPod:
+
+```ruby
+platform :ios, '13.0'
+use_frameworks! # This is important
+
+target 'YourApp' do
+  pod 'SentrySwiftUI'
+end
+```
+
+Or by importing sentry-cocoa repository (https://github.com/getsentry/sentry-cocoa) with SPM.
+
+> If you plan to add _SentrySwiftUI_, you don`t need to also add _Sentry_ library since the former dependents on the latter.
+
+In order to start monitoring the performance of your views, you need to wrap it with `SentryTracedView`, like this:
+```swift {tabTitle:Swift}
+
+import SentrySwiftUI
+
+var body: some View {
+    SentryTracedView("<NAME YOUR TRANSACTION>"){
```

Comment:
`m`: I think we could simply change this to pass in your view name, so users don't have to think about transaction names. They simply put in their view names, and Sentry does the rest. I think we should add a sentence, that the SDK will use the root view name as transaction name. We then would only need one code sample with nested views. Not two.
```suggestion
    SentryTracedView("<Name of your view>"){
```

I think we should also rename the parameter `transactionName` of `SentryTracedView`. I created an issue for that https://github.com/getsentry/sentry-cocoa/issues/2608.